packages/terra-icon/scripts/src/generate-icon/staticIconLowLightTemplate.txt

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
+import React from 'react';
+import classNames from 'classnames';
+import IconBase from '../IconBase';
+
+const SvgIcon = (customProps) => {
+  const attributes = Object.assign({}, customProps);
+  const iconClassNames = classNames(
+    customProps.className,
+  );
```

Comment:
Might not be worth the effort but at this point we could ditch class names entirely and let them pass through with custom props

Reply:
i did think about this, but didn't want to mess with it